nodejs/awsx/lb/targetGroup.ts

```diff
@@ -45,11 +45,15 @@ export abstract class TargetGroup
 
         this.vpc = args.vpc;
 
+        if (!args.targetGroup && args.port === undefined) {
```

Comment:
Is there a reason to check `port` here? I believe it can be undefined if the `targetType` is lambda. I guess the old version of the args didn't actually allow this, but I'm not sure that that was actually correct?

Reply:
Ah, yeah, I was just doing it based off the previous args. I've updated it to remove this